ubuntudesign/landscape.canonical.com
# Fix linting

#### What's this PR do?
Fixes all the linting and makes sure the build is all valid.

#### Where should the reviewer start?
First pull down the branch ```fix-linking``` and run ```npm install``` to make sure all dependancies are installed.

#### How should this be manually tested?
Once setup run ```gulp test``` and ```gulp build``` and check there are no errors with the SCSS linting.

#### Any background context you want to provide?
_n/a_

#### What are the relevant tickets?
_n/a_

#### Screenshots (if appropriate)
_n/a_

#### Questions:
_n/a_
